helm-charts/falcon-kac/values.yaml

```diff
@@ -59,6 +59,11 @@ falcon:
   tags:
   provisioning_token:
 
+# If set to false, KAC ceaces to act as an admission webhook.
```

Comment:
typo: ceaces -- > "ceases"
I think "the container continues to run." is a little unclear as to which container you are referring to.
I would keep the message short and simple
"If set to false, the KAC ValidatingAdmissionWebhook is not installed"

Reply:
Thanks, will adjust!